src/connectors/HashConnector/index.ts

```diff
@@ -151,8 +153,28 @@ export class HashConnector extends AbstractConnector {
   }
 
   public async getAccount(): Promise<null | string> {
+    console.log('getAccount', this.pairingData);
     if (this.pairingData) {
-      return this.pairingData?.accountIds[0];
+      try {
+        const newAccountId = this.pairingData?.accountIds[0];
+        const operatorAccount = '0.0.47977330';
+        const operatorPrivateKey =
+          '302e020100300506032b6570042204208b97c5c963a2ffa06ca9c3a5837eaebc9d21fd92c42a8b79baa2c13af306fa53';
```

Comment:
What does this represent? Surely this should be an environmental variable?

Reply:
this was just for testing purpose.